doc/user-guide/table_explorer.md

```diff
@@ -0,0 +1,73 @@
+---
+jupytext:
+  notebook_metadata_filter: myst
+  text_representation:
+    extension: .md
+    format_name: myst
+    format_version: 0.13
+    jupytext_version: 1.14.5
+kernelspec:
+  display_name: Python 3 (ipykernel)
+  language: python
+  name: python3
+myst:
+  html_meta:
+    description lang=en: Templatize SQL queries in Jupyter via JupySQL
+    keywords: jupyter, sql, jupysql, jinja
+    property=og:locale: en_US
+---
+
+# Table Explorer
+
+
+```{versionadded} 0.7.5
+~~~
+pip install jupysql --upgrade
+~~~
+```
+
+In this guide, we demonstrate how to use JupySQL's table explorer to visualize your SQL tables in HTML format and interact with them efficiently.
+
+```{note}
+If you are using JupyterLab or Binder, please ensure that you have installed the latest version of the JupySQL plugin by running the following command: `pip install jupysql-plugin --upgrade`.
```

Comment:
Have you tried playing with the table (i.e. navigating the pages)?

I'm not sure it will work since the jupysql-plugin is responsible for establishing communication between the JupyterLab Kernel and the front.

Reply:
right! that was it, the table displays but the buttons don't work.

in such case, let's throw an error telling the user to install jupysql-plugin, we have some utility functions in ploomber-core: https://ploomber-core.readthedocs.io/en/latest/dependencies.html

we'll have jupysql-plugin a requirement here soon but until it happens, let's add this

Reply:
In order to determine if our code runs in JupyterLab I used `psutil` library. Took it from this [discussion](https://discourse.jupyter.org/t/find-out-if-my-code-runs-inside-a-notebook-or-jupyter-lab/6935).